styles/globals.css

```diff
@@ -220,3 +220,9 @@ svg:has(> #Mail) {
 .workspace-drop-down > [role="combobox"] {
   max-width: 14rem;
 }
+
+/* TODO: Sort out configuring ShadCN as we've only ever brought in components from it,
+but never configured it. */
+.bg-muted {
```

Comment:
This is for the universal bar styling @isabensusan. The rest of the styling is already there.